static/src/js/util/createSpatialDisplay.js

```diff
@@ -0,0 +1,51 @@
+import SpatialDisplay from '../components/SpatialDisplay/SpatialDisplay'
+
+/**
+ * Returns a string displaying the spatial information.
+ * @param {Object} spatial Object that holds the different spatial areas.
+ * @returns {String} Returns a string formatting the spatial areas into human readable values.
+ */
+export const createSpatialDisplay = (spatial) => {
+  const {
+    boundingBox,
+    circle,
+    line,
+    point,
+    polygon
+  } = spatial
+
+  const sDisplay = new SpatialDisplay()
```

Comment:
Don't import this component into a util file, just take those util methods and create new util files. Then this util and the SpatialDisplay component will import them to use

Reply:
Ok